webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -149,7 +149,7 @@
                             </tr>
                             <tr>
                                 <th scope="row">OS</th>
-                                <td>{{ report.os_name }}</td>
+                                <td>{{ report.os_pretty_version or report.os_name }}</td>
```

Comment:
I have verified that the code works as it is now. I removed the ``os_pretty_version`` field from the report and verified that no error was raised. Writing a test for that would be cumbersome and quite useless because it would just be testing a jinja feature.